logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -229,6 +230,11 @@ interface ConversationRepository {
         role: Conversation.Member.Role
     ): Either<CoreFailure, Unit>
 
+    suspend fun updateChannelAddPermission(
+        conversationId: ConversationId,
+        channelAddPermission: ChannelAddPermission
+    ): Either<CoreFailure, Unit>
```

Comment:
Not a deal breaker, but to avoid having `ConversationRepository` growing further and further, we could have a `ChannelRepository` for channel-specific operations, like this one.